uint8 hand
float64 force_data
float64 theta_data
